tests/stdlib/test_time.py

```diff
@@ -106,12 +106,19 @@ def test_clock(self):
 
     #######################################################
     # ctime
-    @expectedFailure
     def test_ctime(self):
         self.assertCodeExecution("""
             import time
             print(time.ctime())
```

Comment:
This test will be very hard to pass because it depends too much on the time of execution.

Your best bet for this test is to discard some of the precision, like: `print(time.ctime()[:10], time.ctime()[-4:])`

This is considering only the `date` information and discarding the hours. It's still a flaky test because it will fail when it runs at exactly midnight, but it's at least giving us good confidence that the code does what it's supposed to do.

Makes sense?

Reply:
I've also had the same problem, seems right will do it now.